internal/controller/issuers/apply_test.go

```diff
@@ -22,8 +22,8 @@ import (
 	"sync"
 	"testing"
 
-	fuzz "github.com/google/gofuzz"
 	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/randfill"
```

Comment:
Should we use the `fuzz` import alias here also? I prefer consistency in import aliasing.